Devscord.DiscordFramework/Services/DirectMessagesService.cs

```diff
@@ -52,22 +53,17 @@ public async Task<bool> TrySendMessage(ulong userId, string message, MessageType
         public async Task<bool> TrySendEmbedMessage(ulong userId, string title, string description, IEnumerable<KeyValuePair<string, string>> values)
         {
             var embed = this._embedMessagesService.Generate(title, description, values);
-            try
-            {
-                await Server.SendDirectEmbedMessage(userId, embed);
-                Log.Information("Bot sent embed message {messageTitle}", embed.Title);
-                return true;
-            }
-            catch (Exception ex)
-            {
-                Log.Warning(ex.Message, ex);
-                return false;
-            }
+            return await SendEmbedMessage(userId, embed);
         }
 
         public async Task<bool> TrySendEmbedMessage(ulong userId, string title, string description, Dictionary<string, Dictionary<string, string>> values)
         {
             var embed = this._embedMessagesService.Generate(title, description, values);
+            return await SendEmbedMessage(userId, embed);
+        }
+
+        public async Task<bool> SendEmbedMessage(ulong userId, Embed embed)
```

Comment:
to myślę że może być private
```suggestion
        private async Task<bool> TrySendEmbedMessage(ulong userId, Embed embed)
```